portals/publisher/src/main/webapp/source/src/app/components/Apis/Details/Endpoints/Prototype/MockImplEndpoints.jsx

```diff
@@ -9,115 +9,767 @@
+                                            ))}
+                                            <Typography variant='body1'>
+                                                Please Re-generate.
+                                            </Typography>
+                                        </Stack>
+                                    </MUIAlert>
+
+                                )}
+                                <Stack direction='row' mt={2} spacing={2} alignItems='center' justifyContent='center'>
+                                    <Paper
+                                        sx={{
+                                            p: 3,
+                                            width: '100%',
+                                            borderRadius: 2,
+                                            boxShadow: 1,
+                                        }}
                                     >
-                                        {Object.keys(paths[path]).map((method) => {
-                                            return CONSTS.HTTP_METHODS.includes(method) ? (
-                                                <Grid key={`${path}/${method}`} item>
-                                                    <GenericOperation
-                                                        target={path}
-                                                        verb={method}>
-                                                        <MockScriptOperation
-                                                            resourcePath={path}
-                                                            resourceMethod={method}
-                                                            operation={paths[path][method]}
-                                                            updatePaths={updatePaths}
-                                                            paths={paths}
-                                                            mockScripts={mockScripts}
-                                                        />
-                                                    </GenericOperation>
+
+                                        <Stack direction='row' spacing={2}
+                                            alignItems='center' justifyContent='space-between'>
+                                            <div style={{ flex: 1 }}>
+                                                <Typography
+                                                    variant='h6'
+                                                    sx={{
+                                                        fontWeight: 500,
+                                                        mb: 1,
+                                                    }}
+                                                >
+                                                    {mockConfig.useAI
+                                                        ? 'AI-Assisted Mock Server is Ready!'
```

Comment:
Let's have consistent casing for all new Text used here. Either stick to camel case or sentence case etc.